andela/caret-bn-frontend
# Chat with supplier 170514860

#### What does this PR do?
PR to allow a user to chat with  supplier

#### Description of Task to be completed?
- add chat actions
- add private chat component
- add  supplier chat component

#### How should this be manually tested?
- Clone this branch
- run npm i
- run npm run dev
- as user  when you navigate  to single accommodation page , click on chat button to chat 
-  as a supplier under the account menu icon, click on chat link and you will see a chat button
#### Any background context you want to provide?
`N/A`
#### What are the relevant pivotal tracker stories?
[170514860](https://www.pivotaltracker.com/story/show/170514860)
#### Screenshots (if appropriate)
<img width="1438" alt="Screen Shot 2020-01-15 at 10 28 57" src="https://user-images.githubusercontent.com/47029978/72417689-1113ae00-3782-11ea-8eaa-6b80a3f10970.png">
<img width="1438" alt="Screen Shot 2020-01-15 at 10 29 41" src="https://user-images.githubusercontent.com/47029978/72417690-1113ae00-3782-11ea-9603-346b14f960f6.png">

#### Questions:
`N/A`
